graylog2-web-interface/src/components/bootstrap/BootstrapModalForm.jsx

```diff
@@ -18,12 +18,12 @@ class BootstrapModalForm extends Component {
     this.open = this.open.bind(this);
     this.close = this.close.bind(this);
     this._submit = this._submit.bind(this);
-    this._onModalClose = this._onModalClose.bind(this);
+    this._onModalCancel = this._onModalCancel.bind(this);
   }
 
-  _onModalClose() {
-    if (typeof this.props.onModalClose === 'function') {
-      this.props.onModalClose();
+  _onModalCancel() {
+    if (typeof this.props.onCancel === 'function') {
```

Comment:
I think anything else than resetting the form when explicitly dismissing it does not make sense, and I have yet to see an app where explicitly closing a form does not reset it.

In this scenario `onModalClose` is called through the explicit call of `close()` on `BootstrapModalWrapper`.